settings.gradle.kts

```diff
@@ -43,3 +43,4 @@ include(":auth-supabase")
 //include(":component")
 //project(":component").projectDir = File("../component/component")
 
+include(":auth-firebase-macao:jsApp")
```

Comment:
this seems introduced by mistake, remove it.

Reply:
Yes, you are right, included automatically